nextflow.config

```diff
@@ -1,40 +1,149 @@
-profiles {
+/*
+ * -------------------------------------------------
+ *  nf-core/neutronstar Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ * Cluster-specific config options should be saved
+ * in the conf folder and imported under a profile
+ * name here.
+ */
+
+// Global default params, used in configs
+params {
+    // Workflow flags
+    genome = false
+    reads = "data/*{1,2}.fastq.gz"
```

Comment:
I think again that there are quite a few things here that are not used and can be removed..